src/s1/lib/store.js

```diff
@@ -23,7 +23,7 @@ export default class Store {
 	shouldAutoResizeModal = false;
 	manualVendorConsents = new Set(); // vendor-consent management partially automatic and partially manual depending on the consent screen
 	isModalShowing = false;
-	isSaveShowing = false;
+	isSaveShowing = true;
```

Comment:
Hmm I would probably create a new `config.shouldAlwaysShowSave` feature toggle, default it to `false` and pass it in to allow overriding this behavior. 

Ideally we only want the "save" button visible when a user has purposefuly interacted to set their consent. Otherwise, "Save" receives the same UI priority as "Ok, Got it" and users will be _very_ likely to just "Save" with no consent granted. 

It makes sense to offer a feature that enables "Save" with the same priority as "Ok Got it" (ie, it makes sense to want to be able to offer "Save" consistently), but I would not bake that feature in,  and instead I would provide a config to allow turning the feature on as optional.

Reply:
The "save" button should use less vibrant colors and be smaller than the others so that it has less visual emphasis on the button
![Screen Shot 2021-09-29 at 3 09 20 PM](https://user-images.githubusercontent.com/49927525/135355563-e6ac9a69-3d06-4ed6-9dfb-44c6507a31b5.png)